flowauth/backend/flowauth/main.py

```diff
@@ -184,28 +185,28 @@ def create_app(test_config=None):
     if app.config["DEMO_MODE"]:  # Create demo data
         from flowauth.models import make_demodata
 
-        app.before_first_request(make_demodata)
+        with app.app_context():
+            make_demodata()
```

Comment:
I suppose really these and the following shouldn't live here anyway, and the initial setup stuff should be invoked separately from app spinup.